src/bp-forums/core/capabilities.php

```diff
@@ -608,3 +608,17 @@ function bbp_restore_caps_from_wp_roles() {
 		'message' => __( 'Complete!', 'buddyboss' ),
 	);
 }
+
+// Restore capabilities to create topic tags for non-admin organizer
```

Comment:
@ngolwalkar PLease follow the proper function doc block with since tag and with all the necessary info.